patches/aspnetcore/0005-Fix-version-number.patch

```diff
@@ -1,24 +0,0 @@
-From 366568840602e3d7b99e5a3076c24dd9d53dada3 Mon Sep 17 00:00:00 2001
-From: dseefeld <dseefeld@microsoft.com>
-Date: Wed, 30 Sep 2020 14:13:13 +0000
-Subject: [PATCH] Fix version number
-
-This file is actually a very long minified JS file with a version number at the
-end. The file is treated as binary, making this patch less clear, but it's easy
-to update/reapply it with a text editor.
----
- .../Web.JS/dist/Release/blazor.server.js           | Bin 245791 -> 245793 bytes
```

Comment:
Is this just not needed anymore or did the version number move somewhere else?

Reply:
The version number was 5.0.0.

Reply:
I take it you mean that the version number already present in the file is 5.0.0, so we don't need to patch it to 5.0.0? Maybe we should open an issue to track adding this patch back in 5.0.1 when the version gets desynchronized again, so we're less likely to forget/rediscover it in the future?

Reply:
Yes.  It didn't need to be updated.  I added https://github.com/dotnet/source-build/issues/1855